docs/baseband/mark4/index.rst

```diff
@@ -48,7 +50,7 @@ also provide a reference time within 4 years of the observation start time::
     ...                 ref_time=Time('2013:100:23:00:00'))
     >>> fh
     <Mark4StreamReader name=... offset=0
-        frames_per_second=400, samples_per_frame=80000,
+        sample_rate=3.2e+07 Hz, samples_per_frame=80000,
```

Comment:
guess you hadn't gotten to the doc yet...

Reply:
Not sure I understand - isn't that fine (modifications you mentioned above notwithstanding)?

Reply:
Just thought that for Mark4 at least, the sample rate should be in MHz.